javascript/index.js

```diff
@@ -54,12 +68,34 @@ function setResetBtn() {
   // ... your code goes here
 }
 
+let everySecond = setInterval(function () {
+  printTime();
+}, 10);
+
 // Start/Stop Button
 btnLeftElement.addEventListener('click', () => {
-  // ... your code goes here
+  if (btnLeftElement.classList.contains('start')) {
+    btnLeftElement.innerText = 'STOP';
+    btnRightElement.innerText = 'SPLIT';
+    btnLeftElement.classList.toggle('start');
+    btnLeftElement.classList.toggle('stop');
+    chronometer.start();
+  } else {
+    chronometer.stop();
+    btnLeftElement.innerText = 'START';
+    btnRightElement.innerText = 'RESET';
+    btnLeftElement.classList.toggle('stop');
+    btnLeftElement.classList.toggle('start');
+    chronometer.stop();
+  }
 });
```

Comment:
Here you should be extracting logic to separate functions -> code will be easier to read and follow. 
For example:

```
function setStartBtn() {
  btnLeftElement.classList.toggle('start');
  btnLeftElement.innerText = 'START';
}
```

and then call `setStartBtn()` on line 78.
So finally your event listener would have this. And logic is extracted in separate functions

```
btnLeftElement.addEventListener('click', () => {
  if(btnLeftElement.classList.contains('start')) {
    chronometer.start(printTime);
    setStopBtn();
    setSplitBtn();
  } else {
    chronometer.stop();
    setStartBtn();
    setResetBtn();
  }
});
```